documentation/practices.md

```diff
@@ -52,15 +52,13 @@ Catalog images and site content images can be uploaded to WCH using the wchtools
 The React PWA Reference Storefront is pre-configured for integration with the Google Analytics enhanced e-commerce plugin. 
 
 ### Route/Page Views
-The handlers for integrating with Google Analytics allow for routes, which are pre-configured in the `App.js` file, to be logged as pageviews in the sites real-time traffic data. Further configuration in Google Analytics will enable measurements for conversion rates based on the pages viewed in the web traffic. 
-`Analytics.js` contains the functions used for logging page views with ReactGA (React library for invoking Google Analytics functions), and functions for invoking the enhanced e-commerce capabilities.
+In the `App.js` file, integration handlers for Google Analytics are pre-configured to allow for routes to be logged as pageviews in a sites real-time traffic data. Any further measurements for conversion rates based on the page views in the web traffic must be configured in Google Analytics.
+The `Analytics.js` file contains the functions used for logging page views with the React library for invoking Google Analytics functions (ReactGA), and enhanced e-commerce capabilities.
```

Comment:
saying it's "the React library" implies it's the only one. Should be "a React library"